examples/nodejs-express-launchdarkly/lib/arcjet.js

```diff
@@ -0,0 +1,45 @@
+import _arcjet, { shield, slidingWindow } from "@arcjet/node";
+import { getArcjetConfig } from "./launchdarkly.js";
+
+// Initialize Arcjet with your site key and rules
+const aj = _arcjet({
+  // Get your site key from https://app.arcjet.com
+  // and set it as an environment variable rather than hard coding.
+  // See: https://www.npmjs.com/package/dotenv
+  key: process.env.ARCJET_KEY,
+  rules: [],
+});
+
+// We don't want to reinitialize Arcjet unless the configuration changes
+// Let's create a global instance of Arcjet with the rules applied
+// On first load, we'll use the default Arcjet instance without rules
+let ajWithRules = aj;
+
+// We'll store the current LaunchDarkly configuration here
+let currentConfig = {};
+
+// This function will return an Arcjet instance with the latest rules
+const arcjet = async () => {
+  // Get the latest configuration from LaunchDarkly
+  const newConfig = await getArcjetConfig();
```

Comment:
This has to fetch the data each time so there's no point in caching the aj client. It is likely that `JSON.stringify` is going to be more costly than using `withRule`